astro/src/content/docs/extend/code/lambdas/testing.mdx

```diff
@@ -0,0 +1,429 @@
+This command will run FusionAuth and set up a sample application with an API Key and a User, configured in the `kickstart.json` file in the `kickstart` subdirectory.
+
+FusionAuth will be initially configured with these settings.
+
+* Your example username is `richard@example.com` and the password is `password`.
+* Your admin username is `admin@example.com` and the password is `password`.
+* The base URL of FusionAuth [http://localhost:9011/](http://localhost:9011/).
+
+<Aside type="caution">
+  The `.env` and `kickstart.json` files contain passwords. In a real application, always add these files to your `.gitignore` file and never commit secrets to version control.
+</Aside>
+
+## Manually Create A Simple Lambda
+
+Let's start by making a simple lambda to test that it works on your machine.
+
+- Log in to the [FusionAuth admin UI](http://localhost:9011/admin).
+- Navigate to "Customizations -> Lambdas".
+- Click the <IconButton icon="plus" color="green" /> button at the top right to add a new lambda.
+- Enter the Id `f3b3b547-7754-452d-8729-21b50d111505`.
+- Enter the Name `[ATest]` (to put it at the top of the list of lambdas alphabetically).
+- Select the Type `JWT Populate`.
+- Leave the Engine as `GraalJS`.
+- Enable <InlineUIElement>Debug Enabled</InlineUIElement> so that you can see messages in the event log.
+- Add the line `jwt.message = 'Hello World!';` to the body of the `populate` function.
+
+The body should now be similar to below.
+
+```js
+function populate(jwt, user, registration) {
+  jwt.message = 'Hello World!';
+  console.info('Hello World!');
+}
+```
+
+<img src="/img/docs/extend/code/lambdas/populate-lambda.png" alt="Creating a JWT populate lambda" role="bottom-cropped" width="1200" />
+
+Save the lambda.
+
+Now activate the lambda for the example app.
+
+- Navigate to "Applications" and click the <IconButton icon="edit" color="blue" /> button on the "Example app".
+- Click on the "JWT" tab.
+- Toggle <InlineUIElement>Enabled</InlineUIElement> to on.
+- Under "Lambda Settings", select the lambda you created, called "[ATest]", for the `Access Token populate lambda`.
+- Click the <IconButton icon="save" color="blue" /> button to save the changes.
+
+<img src="/img/docs/extend/code/lambdas/edit-application.png" alt="Enabling the lambda in an application" className="img-fluid" width="1200" />
+
+You can now test that the new lambda writes to the event log and returns extra data in the JWT. The repository you downloaded contains two directories.
+- `complete-application` — This is the result of the work you will complete in this guide if you need to refer to the finished files. Do not work in this directory.
+- `app` — This contains a basic application to which you will add tests in this guide. Work in this directory.
+
+Open the `routes/index.js` file in the `app` directory.
+
+Add a line in the OAuth redirect route handler to write the JWT to the console at login. The start of the function should now look like the code below.
```

Comment:
Not super clear where to add this.  Could not that there is a comment `/* OAuth return from FusionAuth */` above the function to update.